examples/rgbpp/xudt/btc-transfer-all/1-btc-transfer-all.ts

```diff
@@ -0,0 +1,75 @@
+import { bitcoin } from 'rgbpp/btc';
+import { buildRgbppTransferAllTxs, sendRgbppTxGroups } from 'rgbpp';
+import { btcDataSource, isMainnet, collector, btcAccount } from '../../env';
+import { signPsbt } from '../../shared/btc-account';
+import { saveCkbVirtualTxResult } from '../../shared/utils';
+
+interface TestParams {
+  xudtTypeArgs: string;
+  fromAddress: string;
+  toAddress: string;
+}
+const SEND_TX_GROUPS = false;
```

Comment:
Could you explain the usage of `SEND_TX_GROUPS`?

Reply:
> Could you explain the usage of `SEND_TX_GROUPS`?

`SEND_TX_GROUPS` is a switch that controls whether transactions are sent. Since batch transfer operations involve transferring all assets to a target address, there is a risk, such as accidentally transferring all assets to the wrong address. When `SEND_TX_GROUPS` is set to `false`, the system will only construct and sign the transactions but will not actually send them. This helps developers verify the transaction details before execution, reducing the risk of mistakes.

Reply:
I see. If developers don't want to send the transactions, they can delete or comment out this line of code `sendRgbppTxGroups`. I don't think the `SEND_TX_GROUPS` is required.